ios/KalturaPlayerViewManager.swift

```diff
@@ -384,6 +403,19 @@ class KalturaPlayerRNView : UIView {
         kalturaPlayer.loadMedia(options: mediaOptions) { error in
             if (error != nil) {
                 print("Error in loadMedia: %@", error!)
+                KalturaPlayerEvents.emitter.sendEvent(withName: "loadMediaFailed", body: [
+                    "code": "ConcurrencyLimitation",
+                    "message": "Concurrency limitation",
+                    "name": "OTTError",
+                    "type": "loadMediaFailed"
+                ])
+            } else {
+                KalturaPlayerEvents.emitter.sendEvent(withName: "loadMediaSuccess", body: [
+                    "payload": [
+                        "id": assetId,
+                        "type": "loadMediaFailed"
```

Comment:
?!?!? loadMediaSuccess --->loadMediaFailed
and also here this is thr same object as we get from Android?
I know we are not getting the same object from `loadMedia` completion block but we have apis in player to get the MediaAsset object.
Please compare android and iOS and expose same data!